dist/sizzle.js

```diff
@@ -782,7 +793,19 @@ setDocument = Sizzle.setDocument = function( node ) {
 	// See https://bugs.jquery.com/ticket/13378
 	rbuggyQSA = [];
 
-	if ( (support.qsa = rnative.test( document.querySelectorAll )) ) {
+	//----------------------------------------------------------
+	/**
+	 * @author Skuid
+	 * In Lightning Locker, post Winter '20, the document.querySelectorAll function is no longer the native
+	 * implementation, but it can still be used as if it were. It is safe to assume that all browsers supported by
+	 * Lightning also support this feature.
+	 * https://releasenotes.docs.salesforce.com/en-us/winter20/release-notes/rn_lwc_dom_api.htm
+	 * 
+	 * This issue is documented in CORE-2571
+	 */
+	support.qsa = rnative.test( document.querySelectorAll ) || !!window.$A;
```

Comment:
If `rnative.test( document.querySelectorAll )` fails, then test if `window.$A` is defined. If it is, then set `support. qsa ` to true.